include/libtorrent/alert_manager.hpp

```diff
@@ -202,7 +209,11 @@ namespace libtorrent {
 
 #ifndef TORRENT_DISABLE_EXTENSIONS
 		typedef std::list<boost::shared_ptr<plugin> > ses_extension_list_t;
+		void notify_extensions(alert * const a, ses_extension_list_t& extensions);
```

Comment:
the ``extensions`` parameter is not mutated by this call, right? It should be a ``const&``